src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ErrorKinds.txt

```diff
@@ -1,11 +1,11 @@
-﻿>> Abs(1/0)
+>> Abs(1/0)
 #Error(Kind=Div0)
 
 >> Abs(If(1/0<2,[1]))
 #Error(Kind=Div0)
 
 >> ForAll(Abs([0, -1, 1/0, 1, If(Char(0) = "a",-1)]), IfError(Value, -FirstError.Kind))
-[0,1,-13,1,-25]
+Table({Value:0},{Value:1},{Value:-13},{Value:1},{Value:-25})
```

Comment:
>Table({Value:0} [](http://example.com/codeflow?start=0&length=15)

ToExpression could emit this as a [ ] ,  that may reduce a significant number of changes. 

Although the consistency of emitting everything as a Table is kind of nice.

Reply:
After offline conversation, it was agreed that ToExpression will emit `Table({Value:0}` format.